diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/LineLength.kt

```diff
@@ -20,6 +47,10 @@ class LineLength : Rule("line-length") {
 
     companion object {
         private const val MAX_LENGTH = 120L
+        private const val LEFT_OFFSET = 4
+        private const val BINARY_OFFSET = 13
```

Comment:
Please provide some documentation for these constants, I can't quite understand their purpose by their names and usage.